src/components/note/__internal__/status-with-tooltip/status.spec.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import StatusWithTooltip from './status.component';
+import Tooltip from '../../../tooltip';
+import { assertStyleMatch } from '../../../../__spec_helper__/test-utils';
+
+const onMouseOver = jest.fn();
+const onMouseLeave = jest.fn();
+
+const render = (props = {}, renderer = mount) => {
```

Comment:
`mount` is the only renderer in this file so you can get rid of this `renderer` parameter and just use `mount below

Reply:
see above reply